core/src/main/java/org/wildfly/extras/sunstone/api/impl/NodeConfigData.java

```diff
@@ -5,13 +5,8 @@
  * data between a cloud-provider-specific subclass and general {@link AbstractJCloudsNode}.
  */
 public final class NodeConfigData {
```

Comment:
I'm thinking about removing the class completely. IMO it's cleaner than to keep it for just-in-case.